lib/dogapi/v1/monitor.rb

```diff
@@ -17,9 +17,12 @@ def monitor(type, query, options = {})
       end
 
       def update_monitor(monitor_id, query, options)
```

Comment:
Actually, nothing is optional here, so we'll need to modify this as well.
Like so 
```suggestion
       def update_monitor(monitor_id, query = nil, options = {})
```